docs/conf.py

```diff
@@ -64,6 +64,25 @@
     'nbsphinx',
     'IPython.sphinxext.ipython_console_highlighting'
 ]
+# Adding jinja passthrough for html
+# Taken from https://www.ericholscher.com/blog/2016/jul/25/integrating-jinja-rst-sphinx/
+html_context = {"release": release,
+                "version": version}
+def rstjinja(app, docname, source):
+    """
+    Render our pages as a jinja template for fancy templating goodness.
+    """
+    # Make sure we're outputting HTML
+    if app.builder.format != 'html':
+        return
+    src = source[0]
+    rendered = app.builder.templates.render_string(
+        src, app.config.html_context
```

Comment:
this `html_context` is the dictionary build on line `69` and will provide the variables that `jinja2` will put in scope during rendering.